src/trackers/classes/crawl.js

```diff
@@ -95,6 +101,29 @@ function _processSite (crawl, site) {
             crawl.commonRequests[key].update(request, site)
         }
     })
+
+    for (const [script, apis] of Object.entries(site.siteData.data.apis.callStats)) {
+        const apisUsed = Object.keys(apis)
+        const tracking = apisUsed.length >= 15 ? true : false
```

Comment:
```suggestion
        const tracking = (apisUsed.length >= API_COUNT_THRESHOLD)
```

Can we export threshold to a separate variable somewhere at the top?